qiskit/synthesis/linear_phase/cnot_phase_synth.py

```diff
@@ -60,18 +60,18 @@ def synth_cnot_phase_aam(cnots, angles, section_size=2):
                  x1^x2 + x0 + x0^x3 + x0^x1^x2 + x0^x1^x3 + x0^x1
 
         angles (list): a list containing all the phase-shift gates which are
-            to be applied, in the same order as in "cnots". A number is
+            to be applied, in the same order as in ``cnots``. A number is
             interpreted as the angle of p(angle), otherwise the elements
-            have to be 't', 'tdg', 's', 'sdg' or 'z'.
+            have to be ``'t'``, ``'tdg'``, ``'s'``, ``'sdg'`` or ``'z'``.
 
-        section_size (int): the size of every section, used in _lwr_cnot_synth(), in the
-            Patel–Markov–Hayes algorithm. section_size must be a factor of num_qubits.
+        section_size (int): the size of every section, used in ``_lwr_cnot_synth()``, in the
```

Comment:
Would it make sense to remove the reference to a private function? Seems like it would be easily missed if we change that function, plus it's probably not very useful for the user to know that this arg is used  there 🙂